src/travix/Travix.hx

```diff
@@ -619,6 +668,9 @@ class Travix {
       case 'php': t.doPhp();
       case 'python': t.doPython();
       case 'cs': t.doCs();
+      case 'lua': 
+        if(haxeVersion < "3.3.0") die('lua requires haxe >= 3.3.0');
```

Comment:
Understandable, although if you can't generate lua code to start with, there's nothing to be run. But you're right, such details would better be discussed when it's working ;)